docs/notebooks/simulation/particle_stepper.ipynb

```diff
@@ -4,10 +4,7 @@
    "cell_type": "code",
```

Comment:
<p>Is it still in need of a rewrite for speed? Maybe! But I think we can take this out</p><p> </p><p>Also, stepper -&gt; Tracker everywhere in the notebook, for consistency </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>